change/@fluentui-react-charting-d8f93860-bbc9-4141-8dd7-2549d2e30dd6.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
## [🕵 fluentuiv8 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=30913&commitId=f5685e211180b5fd40104ef4c65e548536562bd7&env=prod&clientType=FLUENTUI)


  <details><summary><b>react-charting-AreaChart</b> 4 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|react-charting-AreaChart.Basic - Dark Mode.hover.chromium.png|1746| Changed|
|react-charting-AreaChart.Multiple - Dark Mode.hover.chromium.png|3965| Changed|
|react-charting-AreaChart.Basic.hover.chromium.png|3079| Changed|
|react-charting-AreaChart.Multiple.hover.chromium.png|4583| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v8"/>